physicalplan/plan.go

```diff
@@ -122,6 +134,22 @@ func newOperator(expr parser.Expr, storage storage.Queryable, mint time.Time, ma
 	}
 }
 
+func aggregateValidation(node parser.Node, want parser.ValueType) error {
```

Comment:
Hm should we do this validation when the aggregator is instantiated?

Reply:
good point.